shared/chat/conversation/input-area/container.js

```diff
@@ -11,6 +11,7 @@ type OwnProps = {|
   focusInputCounter: number,
   isPending: boolean,
   onScrollDown: () => void,
+  onScrollUp: () => void,
```

Comment:
This ideally shouldn't be called `onScrollUp` I think. `onX` implies that this will be called when `X` happens. But that's not the case here. Here it is called when we want `X` to happen. This should probably be called `onRequestScrollX`. It's a medium size change and it isn't super important so 🐑

Reply:
Done!